mod/wikirate/set/right/general_overview.rb

```diff
@@ -0,0 +1,55 @@
+format :html do
+  view :missing do |args|
+    handle_edit_general_overview(args) { super args }
+  end
+
+  view :content do |args|
+    handle_edit_general_overview( args ) { super args }
+  end
+
+  view :titled_with_edits do |args|
+    handle_edit_general_overview( args ) { super args }
+  end
+
+  view :editor do |args|
+    prompt = with_inclusion_mode :normal do
+      if claim_name = params[:citable] && claim = Card[claim_name]
```

Comment:
Assignment in condition - you probably meant to use `==`.<br>Useless assignment to variable - `claim_name`. Did you mean `claim`?